python/cairo-addons/src/cairo_addons/testing/runner.py

```diff
@@ -461,7 +461,7 @@ def run_rust_vm(
     request: FixtureRequest,
     coverage: Optional[Callable[[pl.DataFrame, int], pl.DataFrame]],
 ):
-    def _run(entrypoint, *args, **kwargs):
+    def _run(entrypoint, check_squashed_dicts: bool = False, *args, **kwargs):
```

Comment:
can we keep the same name, either `check_` in arg and in runner attribute, or `verify_` in both